### PR TITLE
Add CONTRIBUTING.md and update README.md

### PR DESCRIPTION
**Pull request by pieths-ms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,21 +121,23 @@ As the maintainer of this project, please make a few updates:
...
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+We welcome contributions and suggestions! See `CONTRIBUTING.md` for more details.
+
+### Issues and feedback
+
+All forms of feedback are welcome through this repo's issues:
+https://github.com/Azure/automl-devplat2-preview/issues
```
Comment (CESARDELATORRE) [inline README.md]: This needs to be ISSUES from https://github.com/Azure/automl-testdataset-preview, not DevPlat2.0 PREVIEW site.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,103 @@
...
+# Get test predictions
+test_run_details = test_run.get_details()
+test_run_predictions = Dataset.get_by_id(workspace, test_run_details['outputDatasets'][0]['identifier']['savedId'])
+test_run_predictions.to_pandas_dataframe().head()
+```
+
+`ModelProxy` already returns the predictions and metrics and
+does not require further processing to retrieve the outputs.
 
 ## Requirements
```
Comment (CESARDELATORRE) [inline README.md]: Is this true? "This datastore is visible to all users with the same subscription. Do not use this feature if any of the information used for or created by the test run needs to remain private."

The datastore and specifically Blog Storage under the covers is secured so ONLY the people with access to the WORKSPACE can have access to it, but not *ANY* user with the same Azure subscription, right?

File: README.md [hunk 3]
```diff
@@ -29,21 +121,23 @@ As the maintainer of this project, please make a few updates:
 - `test_data`/`test_size` is not supported at the same time as `cv_split_column_names`
 - `test_data`/`test_size` is not supported at the same time as `cv_splits_indices`
 - Test runs are not supported for AutoML runs which have streaming enabled.
```
Comment (CESARDELATORRE) [inline README.md]: I'm not sure I was aware of this: "Test runs are not supported for AutoML runs which have streaming enabled".
Why is that? Why models/algos (Based on NimbusML I guess) using data streaming do not support it?

Also, users don't really "Enable streaming", there's no such parameter in AutoMLConfig class, so if this is happening we need to be more specific/explicit and list the models/algos not supported by Model-Test. Right?

File: README.md [hunk 4]
```diff
@@ -1,11 +1,103 @@
...
+2. Test any model after the main AutoML run has completed ("on-demand" model testing).
+   These test runs are started manually by the user either using the SDK or the UI.
+
+Model testing is supported for the following task types:
+
+- Classification
+- Regression
+- Forecasting
+
+There are three ways to start a test run from a script or a notebook.
```
Comment (CESARDELATORRE) [inline README.md:42]: Please add a title here like "Quick Walkthrough" or "Getting Started".

But before this section, is there anything about SETUP that a user needs to do?

For instance, requirements, version needed, etc. any pre-requisite should come first, before the "Getting Started".

"Limitations" is okay to have it later, as you have it.
But we need the "pre-requisites" up-front with a Title like "Requirements" or "Pre-requisites" and then a few lines explaining it.  :)
Even related to the Azure Subscription, Workspace requisite, etc. Anything that is needed before trying this feature. :)

File: README.md (hunk 3 above)
Comment (pieths-ms) [reply README.md:135]: Updated the docs to mention how the workspaces and resource groups can be made more secure.

File: README.md (hunk 2 above)
Comment (pieths-ms) [reply README.md]: Updated the docs to mention how the workspaces and resource groups can be secured using IAM.

File: README.md (hunk 3 above)
Comment (pieths-ms) [reply README.md]: Updated the wording to only mention the model selection criteria which are used to determine if a run will be a streaming run.

**Review by CESARDELATORRE — APPROVED** [review]
LGTM! :)

**Pull request merged**